allink_core/core_apps/allink_content/cms_plugins.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, *args, **kwargs):
                 widget=forms.CheckboxSelectMultiple(),
                 label='Predifined variations',
                 choices=get_additional_choices('CONTENT_CSS_CLASSES'),
+                initial=get_additional_choices('INITIAL_CONTENT_CSS_CLASSES'),
```

Comment:
could we make get_additional_choices more sophisticated, so that we would not have to set a tuple in settings?

Reply:
i know where you want me to go but this doesn't work as django at a later point receives CONTENT_CSS_CLASSES and expects only two values per tuple